File: src/TmrOverlay.App/Overlays/GarageCover/GarageCoverOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.GarageCover;

internal static class GarageCoverOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "garage-cover",
        DisplayName: "Garage Cover",
        DefaultWidth: 1280,
        DefaultHeight: 720,
        ShowScaleControl: false,
        ShowOpacityControl: false);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/GarageCover/GarageCoverOverlayDefinition.swift
import AppKit

enum GarageCoverOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "garage-cover",
        displayName: "Garage Cover",
        defaultSize: NSSize(width: 1280, height: 720),
        showScaleControl: true,
        showOpacityControl: false
    )
}
